bin/brew

```diff
@@ -109,6 +109,16 @@ then
   export HOMEBREW_EDITOR="${VISUAL}"
 fi
 
+# set from user environment
+if [[ -z "${HOMEBREW_PIP_INDEX_URL}" && -n "${PIP_INDEX_URL}" ]]
```

Comment:
I don't think we should set this automatically.

Reply:
Do you mean the user should explicitly set a Homebrew-specific variable `HOMEBREW_PIP_INDEX_URL`?

If the user sets a mirror:

```bash
export PIP_INDEX_URL="https://pypi.tuna.tsinghua.edu.cn/simple"
```

All `pip` sdist/wheels will download from `https://pypi.tuna.tsinghua.edu.cn/package`:

```bash
pip3 install some-package
```

Reply:
Yeh, I'm proposing a single Homebrew variable, not space separated.